settings.gradle.kts

```
@@ -16,6 +16,7 @@ dependencyResolutionManagement {
         google()
         mavenCentral()
         maven(url = "https://plugins.gradle.org/m2/")
+        maven(url = "https://oss.sonatype.org/content/repositories/snapshots/")
```

Maybe limit it to only allow Stream artifacts from this snapthots repository? :thinking: 
```suggestion
        maven(url = "https://oss.sonatype.org/content/repositories/snapshots/") {
			content {
				includeGroup("io.getstream")
			}
		}
```

---

We should use snapshots only for our snapshot/alpha version. If not, customers also need to add this repository on their Maven Central configuration.

---

Yep, it is only for our internal development when we need a pre-release version to complete a task